src/Bundle/FrameworkBundle/fixtures/BarCommand.js

```diff
@@ -0,0 +1,21 @@
+const Command = Jymfony.Component.Console.Command.Command;
+
+class BarCommand extends Command {
+    configure() {
+        this.name = 'command:bar';
+        this.description = 'The command:bar command';
+    }
+
+    * interact(input, output) {
```

Comment:
That's a good point. Said that, other fixtures command should be fixed this way.